DToCToDB.md

```diff
@@ -0,0 +1,1193 @@
+      <th>Local Authority Code</th>
+      <th>Local Authority Name</th>
+      <th>Acute Or Non Acute Description</th>
+      <th>Reason For Delay</th>
+      <th>NHS A SUM</th>
+      <th>NHS B SUM</th>
+      <th>Social Care A SUM</th>
+      <th>Social Care B SUM</th>
+      <th>Both A SUM</th>
+      <th>Both B SUM</th>
+      <th>Date</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>0</th>
+      <td>Q82</td>
+      <td>NHS ENGLAND SOUTH (SOUTH CENTRAL)</td>
+      <td>AXG</td>
+      <td>WILTSHIRE HEALTH &amp; CARE LLP</td>
+      <td>817</td>
+      <td>WILTSHIRE</td>
+      <td>Non-Acute</td>
+      <td>I_HOUSING</td>
+      <td>0.0</td>
+      <td>6.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>2016-10-01</td>
+    </tr>
+    <tr>
+      <th>1</th>
+      <td>Q82</td>
+      <td>NHS ENGLAND SOUTH (SOUTH CENTRAL)</td>
+      <td>AXG</td>
+      <td>WILTSHIRE HEALTH &amp; CARE LLP</td>
+      <td>817</td>
+      <td>WILTSHIRE</td>
+      <td>Non-Acute</td>
+      <td>H_DISPUTES</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>2016-10-01</td>
+    </tr>
+    <tr>
+      <th>2</th>
+      <td>Q82</td>
+      <td>NHS ENGLAND SOUTH (SOUTH CENTRAL)</td>
+      <td>AXG</td>
+      <td>WILTSHIRE HEALTH &amp; CARE LLP</td>
+      <td>817</td>
+      <td>WILTSHIRE</td>
+      <td>Non-Acute</td>
+      <td>G_PATIENT_FAMILY_CHOICE</td>
+      <td>3.0</td>
+      <td>29.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>2016-10-01</td>
+    </tr>
+    <tr>
+      <th>3</th>
+      <td>Q82</td>
+      <td>NHS ENGLAND SOUTH (SOUTH CENTRAL)</td>
+      <td>AXG</td>
+      <td>WILTSHIRE HEALTH &amp; CARE LLP</td>
+      <td>817</td>
+      <td>WILTSHIRE</td>
+      <td>Non-Acute</td>
+      <td>F_COMMUNITY_EQUIP_ADAPT</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>2016-10-01</td>
+    </tr>
+    <tr>
+      <th>4</th>
+      <td>Q82</td>
+      <td>NHS ENGLAND SOUTH (SOUTH CENTRAL)</td>
+      <td>AXG</td>
+      <td>WILTSHIRE HEALTH &amp; CARE LLP</td>
+      <td>817</td>
+      <td>WILTSHIRE</td>
+      <td>Non-Acute</td>
+      <td>E_CARE_PACKAGE_IN_HOME</td>
+      <td>6.0</td>
+      <td>224.0</td>
+      <td>8.0</td>
+      <td>163.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>2016-10-01</td>
+    </tr>
+  </tbody>
+</table>
+</div>
+
+
+
+The dataframe is almost ready to be exported to SQLite, however currently the column names are inappropriate and don't match [SQL naming conventions](https://launchbylunch.com/posts/2014/Feb/16/sql-naming-conventions/). Therefore we are going to rename all the columns to something more appropriate for a SQL database.
+
+
+```python
+dtoc = dtoc.rename(columns={
+        'Provider Parent Org Code':'area_team_code',
```

Comment:
If you aren't too worried about the names and want to simplify this code, you could just lowercase everything and replace spaces with underscores